### PR TITLE
Integration with ODIM's services part #7

### PR DESCRIPTION
**Pull request by MaciejMis**
Changes:
- Request logger expanded,
- Reset action implemented for ComputerSystem

### CODE REVIEW

File: src/rest/loggingHandler.go [hunk 1]
```diff
@@ -10,7 +13,22 @@ type loggingHandler struct {
...
+	// Retrieve request information from a request.
+	err := ctx.ReadJSON(&reqInfo)
+	if err != nil {
+		missingInfoError := "Unable to retrieve information from a request: " + err.Error()
+		logrus.Warn(missingInfoError)
+
+		ctx.Next()
+	}
+
+	requestUri := utils.UriTranslator.DmToRedfish(fmt.Sprintf("https://%s%s", reqInfo.Host, ctx.Request().RequestURI))
```
Comment (ehaligow) [inline src/rest/loggingHandler.go:30]: There is no var UriTranslator any more, need to change to UriConverter.

File: src/rest/postResetSystem.go [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+		logrus.Error(missingInfoError)
+		ctx.StatusCode(http.StatusBadRequest)
+		ctx.WriteString(missingInfoError)
+		return
+	}
+
+	httpClient := redfish.NewHttpClient(r.cfg).WithBasicAuth(reqInfo.Username, string(reqInfo.Password))
+	systemsUri := fmt.Sprintf("https://%s%s", reqInfo.Host, ctx.Request().RequestURI)
+
+	response, err := httpClient.Post(systemsUri, reqInfo.Body)
```
Comment (ehaligow) [inline src/rest/postResetSystem.go:42]: Maybe we can validate the ResetType sending in POST body?

File: src/rest/postResetSystem.go [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+	if err != nil {
+		errorMessage := fmt.Sprintf("POST action failed on %s due to: %s", systemsUri, err.Error())
+		logrus.Error(errorMessage)
+		if response == nil {
+			ctx.StatusCode(http.StatusInternalServerError)
+			ctx.WriteString(errorMessage)
+			return
+		}
+	}
+
```
Comment (ehaligow) [inline src/rest/postResetSystem.go:52]: System service returns error during resetting System after calling DM plugin - the status code Accepted is not accepted by the service.

File: src/rest/loggingHandler.go (hunk 1 above)
Comment (MaciejMis) [reply src/rest/loggingHandler.go]: Fixed

File: src/rest/postResetSystem.go (hunk 2 above)
Comment (MaciejMis) [reply src/rest/postResetSystem.go:52]: Validation added

File: src/rest/postResetSystem.go (hunk 3 above)
Comment (MaciejMis) [reply src/rest/postResetSystem.go:62]: Fixed

**Pull request merged**